lib/src/theme/flutter_deck_text_theme.dart

```diff
@@ -4,6 +4,10 @@ import 'package:flutter/widgets.dart';
 class FlutterDeckTextTheme {
   /// Creates a theme to style text in a slide deck.
   const FlutterDeckTextTheme({
+    this.headerLarge = const TextStyle(
```

Comment:
We are using Material Design 3 font sizing values there, so you can rename this property to `display` with the `fontSize` = 91.